packages/storybook-utils/src/actions.ts

```diff
@@ -9,6 +10,8 @@ import type { DefineStorybookActionsAndVModelsOptions, ExtractVueEventNames } fr
  * the given events as well as implementing v-model handlers so that the Storybook controls are updated when you interact with the component.
  * Should be preferred over manually defining argTypes for *.stories.ts files.
  *
+ * @deprecated Can be replaced globally by using the `withGlobalVModelDecorator` decorator and the `enhanceEventArgTypes` argTypesEnhancer.
```

Comment:
Since we are still beta, why don't we just remove it as breaking change?

Reply:
Okay